### PR TITLE
Feat/app/carousel [MARXAN-517]

### PR DESCRIPTION
**Pull request by anamontiaga**
### Overview

Storybook Carousel component
Home carousel

### Designs

https://www.figma.com/file/hq0BZNB9fzyFSbEUgQIHdK/Marxan-Visual_V02?node-id=2991%3A68

### Feature relevant tickets

https://vizzuality.atlassian.net/secure/RapidBoard.jspa?rapidView=38&projectKey=MARXAN&modal=detail&selectedIssue=MARXAN-517&quickFilter=71

### CODE REVIEW

File: app/tsconfig.json [hunk 1]
```diff
@@ -18,6 +18,7 @@
     "resolveJsonModule": true,
     "isolatedModules": true,
     "jsx": "preserve",
+    "downlevelIteration": true,
```
Comment (anamontiaga) [inline app/tsconfig.json:21]: Added this to manage interation on carousel slider state

File: app/public/styles/tailwind.storybook.css [hunk 2]
```diff
@@ -11396,10 +11433,6 @@ input[type='search']::-webkit-search-decoration,
   line-height: 2;
 }
 
-.leading-extra-loose {
```
Comment (anamontiaga) [inline app/public/styles/tailwind.storybook.css:11399]: Probably because i updated node-modules.

File: app/tsconfig.json (hunk 1 above)
Comment (mbarrenechea) [reply app/tsconfig.json:21]: ok

**Review by mbarrenechea — CHANGES_REQUESTED** [review]
Everything seems to be ok. I made two commits to improve a couple of things:
- https://github.com/Vizzuality/marxan-cloud/pull/325/commits/633b61b6380ac055e0eec623b1ea71075a3747f5
- https://github.com/Vizzuality/marxan-cloud/pull/325/commits/fa39484f45caeb6e89d1006d49c3e4613e221fb4

To approve this PR I need you to implement autoplay behavior. They have an example for doing this => https://codesandbox.io/s/github/rcbyr/keen-slider-sandboxes/tree/master/misc/autoplay/react?file=/src/App.js

File: app/components/carousel/component.tsx [hunk 3]
```diff
@@ -15,18 +15,47 @@ export interface CarouselProps {
...
+      setPause(false);
+    });
+  }, [sliderRef]);
+
+  useEffect(() => {
+    timer.current = setInterval(() => {
+      if (!pause && slider) {
+        slider.next();
+      }
+    }, 2000);
```
Comment (mbarrenechea) [inline app/components/carousel/component.tsx:43]:
```suggestion
    }, 4000);
```

**Review by mbarrenechea — APPROVED** [review]

**Pull request merged**